src/redux/reducers/inputReducer.js

```diff
@@ -18,7 +23,14 @@ export default function inputReducer(state = initialState, action) {
           (task) => task.id !== action.payload.id
         ),
       };
-
+    case MOVE_TASK_TO_IN_PROGRESS:
+      return {
+        ...state,
```

Comment:
use the same approache with "...state" in other cases
"...state.tasksToDo" is wrong